app/config/GuiceModule.scala

```diff
@@ -69,9 +69,18 @@ class GuiceModule(
       servicesConfig.baseUrl("vmv")
     )
 
+    bindCheckFrontendEndpoints()
     bindVmvFrontendEndpoints()
   }
 
+  private def bindCheckFrontendEndpoints(): Unit =
+    bindEndpoints(
+      Map(
+        "check.summary.url" -> "resources.check.frontend.summary.path"
+      ),
+      servicesConfig.getString("business-rates-check.url")
```

Comment:
this will get the check BACKEND service base url, no? https://github.com/hmrc/app-config-base/blob/main/voa-property-linking-frontend.conf#L77
I wouldn't be surprised if this worked fine locally but failed to start up in Dev/QA, can you double-check the config?

Reply:
Good thinking, I'll have a look at the config now.